src/Files.App/UserControls/StatusCenter.xaml

```diff
@@ -740,8 +68,314 @@
 				</Style>
 			</ListView.ItemContainerStyle>
 
+			<ListView.ItemTemplate>
+				<DataTemplate x:DataType="statuscenter:StatusCenterItem">
+					<Grid
+						x:Name="ContentRoot"
+						Padding="8,8,8,8"
+						HorizontalAlignment="Stretch"
+						AutomationProperties.Name="{x:Bind Header, Mode=OneWay}"
+						Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+						BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
+						BorderThickness="1"
+						CornerRadius="{StaticResource OverlayCornerRadius}">
+						<Grid.RowDefinitions>
+							<RowDefinition Height="Auto" />
+							<RowDefinition Height="Auto" />
+						</Grid.RowDefinitions>
+
+						<Grid.Transitions>
+							<TransitionCollection>
+								<RepositionThemeTransition />
+							</TransitionCollection>
+						</Grid.Transitions>
+
+						<!--  Header  -->
+						<Grid x:Name="StatusCenterItemHeaderPanel" Grid.Row="0">
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="Auto" />
+							</Grid.ColumnDefinitions>
+
+							<!--  Header Icon  -->
+							<Grid Grid.Column="0">
+
+								<!--  Circle Background 32x32  -->
+								<Border
+									x:Name="SeverityBackgroundBorder"
+									Width="32"
+									Height="32"
+									HorizontalAlignment="Center"
+									VerticalAlignment="Center"
+									Background="{x:Bind ItemKind, Converter={StaticResource StatusCenterStateToBrushConverter}, ConverterParameter=True, Mode=OneWay}"
+									CornerRadius="16"
+									Opacity="{x:Bind IconBackgroundCircleBorderOpacity, Mode=OneWay}" />
```

Comment:
You can use an icon for this.
![image](https://github.com/files-community/Files/assets/65828559/62b609fe-210a-4ada-8ce0-60c8972456cc)

Reply:
We have decided to use the Border to avoid confusion.

Reply:
I don't understand